composer.json

```diff
@@ -24,7 +24,7 @@
         "issues": "https://github.com/RusticiSoftware/TinCanPHP/issues"
     },
     "require": {
-        "php": "~5.4",
+        "php": "~5.5 || ^7.0",
```

Comment:
Sorry, I missed the point before that you planned to increase the version when this one is released.